src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -749,6 +860,15 @@ components:
         - dispatched
         - not-delivered
 
+    SearchBy:
+      title: SearchBy
+      description: An enumeration of search by fields
+      type: string
+      enum:
+        - surname
+        - applicationReference
+        - electoralRollNumber
```

Comment:
Aside from the fact it might be best to exclude this enum altogether until we get to EIP1-5591, we might also want to include one at a time (i.e. start with just surname).

Reply:
I think keeping these enum now is fine when it's evident that these will be surely supported later on. If ui tries to search by electoral roll and backend hasnt implemented it then it will just return empty list. 

But if i am not too precious about it.